common/types/transaction.go

```diff
@@ -118,6 +118,16 @@ func (t *Transaction) ID() TransactionID {
 	return id
 }
 
+// GetFee returns the fee of the transaction.
+func (t *Transaction) GetFee() uint64 {
+	return t.Fee
+}
+
+// GetRecipient returns the transaction recipient.
+func (t *Transaction) GetRecipient() Address {
+	return t.Recipient
+}
```

Comment:
what's the reason for these getters? seem redundant since Fee and Recipient are meant to be public

Reply:
This is in preparation for removing the fields and deprecating direct access, further down the road in SVM integration